gwardia/templates/gwardia/index.html

```diff
@@ -6,23 +6,57 @@
     <div class="flex flex-col items-center">
         <h1 class="text-4xl text-white font-bold mt-16 mb-8">Panel Gwardii! WIP!</h1>
         <p class="italic text-xs text-gray-500 mb-2">Kliknij w dane spotkanie, by wyświetlić jego opis</p>
-        <ul class="w-1/2" id="js-next-meetings-container">
-            {% for meeting in upcoming_five_meetings %}
-                <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer">
-                    <p class="w-full">
-                        <span class="break-all">{{ meeting.title }}</span>
-                        <span class="float-right">{{ meeting.date_of_meeting }}</span>
-                    </p>
-                    <p class="js-meeting-description border-gray-500 p-5 w-full break-all border-t-2 hidden">
-                        {{ meeting.description|linebreaksbr }}
-                    </p>
-                </li>
-            {% endfor %}
-        </ul>
+        <div  class="w-1/2" id="js-next-meetings-container">
+            <h2 class="text-2xl text-white text-center font-bold mt-8 mb-4">Nadchodzące spotkania</h2>
+            <ul>
+                {% if upcoming_meetings %}
+                    {% for meeting in upcoming_meetings %}
+                        <li class="js-meeting-container flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl cursor-pointer">
+                            <p class="w-full">
+                                <span class="break-all">{{ meeting.title }}</span>
+                                <span class="float-right">{{ meeting.date_of_meeting }}</span>
+                            </p>
+                            <p class="js-meeting-description border-gray-500 p-5 w-full break-word border-t-2 hidden">
+                                {{ meeting.description|linebreaksbr }}
+                            </p>
+                        </li>
+                    {% endfor %}
+                {% else %}
+                    <li class="flex flex-col items-center border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl">
+                        <p class="w-full text-center">Brak nadchodzących spotkań</p>
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+        <div class="w-1/2 hidden" id="js-archival-meetings-container">
```

Comment:
Now we have same code copied 2 times (except archival meetings don't have nice description wgen list is empty). Maybe extract it to component ([inclusion tag](https://docs.djangoproject.com/en/dev/howto/custom-template-tags/#inclusion-tags))